mockserver_client/mockserver_client.py

```diff
@@ -963,7 +965,12 @@ def safe_string_for_file_path(s: str) -> str:
         s = re.sub(r"[^a-z0-9_+]", "", s)
 
         # Limit length if needed
-        max_length = 255  # Most file systems have a 255-character limit for file names
+        max_length: int = (
+            cls.MAX_FILENAME_LENGTH
+        )  # Most file systems have a 255-character limit for file names
+        max_length = (
+            max_length - 3 - 1 - 5
+        )  # subtract 3 characters for index, 1 for "-" and 5 for ".json"
```

Comment:
what if index value is more than one digit? then it could "[10]" or even bigger, then the length for the index could be 4 or bigger than 3?
then this magic number 3 won't work, right?
UNLESS there is a guarantee that this index value is always one digit.